CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -189,7 +189,7 @@ static dispatch_queue_t __ ## PREFIX ## Queue(void) {			\
 #define CF_RETAIN_BALANCED_ELSEWHERE(obj, identified_location) do { } while (0)
 #endif
 
-#if (TARGET_OS_LINUX && !TARGET_OS_ANDROID && !TARGET_OS_CYGWIN) || TARGET_OS_WIN32
+#if (TARGET_OS_LINUX && !TARGET_OS_ANDROID && !TARGET_OS_CYGWIN && defined(__GLIBC__)) || TARGET_OS_WIN32
```

Comment:
Seems like `defined(__GLIBC__) || TARGET_OS_WIN32`?

Reply:
Because Swift on Android is using `import Glibc` I'm confused how exactly their platform checks should look, even in C/C++. cc @finagolfin

Reply:
This C constant works as expected, so it is undefined on non-Glibc platforms like Android. The issue is only in Swift, where [all non-Darwin and non-Windows platforms like Android and the BSDs currently use `import Glibc` to import their C libraries](https://github.com/finagolfin/swift-android-sdk/#porting-swift-packages-to-android) that one has to be careful.